Form data allow null value (#1796)
cfug/dio: dio/CHANGELOG.md

## Unreleased

> - Fix `FormData.fromMap` that did not send the entry if the value was null.

Reviewer: ```suggestion
- Allow `FormData` to send a null entry value as an empty string.
```

  for `ListFormat.multi` and `ListFormat.multiCompatible` since Dio 4.0.x.
- Respect `Options.listFormat` when encoding `x-www-url-encoded` content.

> 

Reviewer: Please revert this.
